### PR TITLE
behnaz/fx-lp-v1

### PR DESCRIPTION
**Pull request by behnaz-fs**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/landing/trade-forex/components/_icon-text-row.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    flex-grow: 0;
+    width: 10rem;
+`
+
+const Row = styled.div`
+    display: flex;
+    flex-direction: row;
```
[the remark this answers is not in the document]
Comment (behnaz-fs) [reply src/pages/landing/trade-forex/components/_icon-text-row.js:21]: sure removing

File: src/pages/landing/trade-forex/components/_image-text-switching.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                    mb="1rem"
+                    weight="bold"
+                >
+                    {localize('Trade forex with ultra-low spreads in 3 simple steps:')}
+                </StyledText>
+
+                {P2P.map((item, index) => {
+                    let is_even = isIndexEven(index, reverse)
+                    return (
+                        <Row flex_direction={!is_even ? 'row' : 'row-reverse'} key={index}>
```
[the remark this answers is not in the document]
Comment (behnaz-fs) [reply src/pages/landing/trade-forex/components/_image-text-switching.js:115]: done

File: src/pages/landing/trade-forex/components/_icon-text-row.js [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+                                {localize(
+                                    'Trade with no additional charges on deposits, withdrawals, and trades.',
+                                )}
+                            </StyledText>
+                        </Column>
+                    </Row>
+                </Card>
+                <Card direction="column">
+                    <Row>
+                        <img src={CompleteOrder} alt="" width="109px" />
```
Comment (habib-deriv) [inline src/pages/landing/trade-forex/components/_icon-text-row.js:173]: Most of the images alt tag is blank. Please add relevent alt tag content

**Review by habib-deriv — COMMENTED** [review]
* A couple of issues, most of the keys are index, let's change this
* The images alt tag are blank

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**